src/controllers/SubscriberController.ts

```diff
@@ -191,6 +191,7 @@ export class SubscriberController {
      * - ready
```

Comment:
_(I still don't know how to review on expanded code, google is not helping)_

I feel the description is confusing now, the listener is returning events that are indeed triggered by those states but only available from a `getState` call and not actually related to all of those events.

Something along those line might be better ?

> Listener on connector events, this listener will get triggered when a connector emits one of those events
> * `reloadRequired`
> * `authChanged`
> * `unloaded`
> * `stateChanged`

Maybe adding `stateChanged - this event will be triggered by connector states: loading, loaded, running, ready, error` ?

And since WRS does not seem to use `onConnectorStateChanged` directly maybe it would be good to rename it `onConnectorChanged` ? seems like it's a switching connectors stream. `onConnectorEvents` ? but the `changed` suffix seems mandatory. Any thought on that ?

It's just that we have `stateChanged` as an event type and the listener is called `onConnectorStateChanged` which is accurate but confusing, we kind of expect to get those loading, loaded... states instead of the only 4 events.

Reply:
I first also wanted to go with `onConnectorEvents` but since it would be the only one not ending on `Changed` I decided to not do it. @brapoprod any opinion on the naming here? I feel like `onConnectorEvents` makes more sense.

Reply:
It should describe what it is, if it makes more sense to diverge from what is there in other calls, that makes sense if the content is different. So fine f me :)